src/test/java/com/gfl/client/util/file/FileJsonParserTest.java

```diff
@@ -0,0 +1,41 @@
+package com.gfl.client.util.file;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.gfl.client.model.ProxyConfigHolder;
+import com.gfl.client.model.ProxyNetworkConfig;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.springframework.core.io.DefaultResourceLoader;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Queue;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
FileNotFoundException: class path resource [json/ProxyConfigs.json] cannot be opened because it does not exist